### PR TITLE
add koniocellular cell

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/cell-ontology/issues/17

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -28486,6 +28487,15 @@ SubClassOf(obo:CL_4023181 obo:CL_0000710)
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002139))
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015658))
 
+# Class: obo:CL_4023187 (koniocellular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Lateral_geniculate_nucleus&oldid=1098375993"^^xsd:string) obo:IAO_0000115 obo:CL_4023187 "A neuron with a small cell body that is located in the koniocellular layer of the lateral geniculate nucleus (LGN).")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023187 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023187 "NIFEXT:44")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) rdfs:comment obo:CL_4023187 "In adult monkeys, koniocellular cells can be distinguished from parvocellular and magnocellular of the LGN by their expression of Calbindin, the alpha subunit of type II calmodulin-dependent protein kinase, and the gamma subunit of protein kinase C.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:28495]: "calbindin" should not be capitalised.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -28486,6 +28487,15 @@ SubClassOf(obo:CL_4023181 obo:CL_0000710)
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002139))
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015658))
 
+# Class: obo:CL_4023187 (koniocellular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Lateral_geniculate_nucleus&oldid=1098375993"^^xsd:string) obo:IAO_0000115 obo:CL_4023187 "A neuron with a small cell body that is located in the koniocellular layer of the lateral geniculate nucleus (LGN).")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:28492]: Not sure about all species, but I think there are several layers of K cells in the LGN. Wording of the definition suggests that there is one layer. 
"... located in the koniocellular layer ..."
Perhaps substituting "the" with "a" would be appropriate.

Also consider adding "K cell" as an exact synonym.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -28486,6 +28487,16 @@ SubClassOf(obo:CL_4023181 obo:CL_0000710)
...
 
+# Class: obo:CL_4023187 (koniocellular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Lateral_geniculate_nucleus&oldid=1098375993"^^xsd:string) obo:IAO_0000115 obo:CL_4023187 "A neuron with a small cell body that is located in a koniocellular layer of the lateral geniculate nucleus (LGN).")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023187 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023187 "NIFEXT:44")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4023187 "K cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) rdfs:comment obo:CL_4023187 "In adult monkeys, koniocellular cells can be distinguished from parvocellular and magnocellular of the LGN by their expression of calbindin, the alpha subunit of type II calmodulin-dependent protein kinase, and the gamma subunit of protein kinase C.")
+AnnotationAssertion(rdfs:label obo:CL_4023187 "koniocellular cell")
+EquivalentClasses(obo:CL_4023187 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0013615)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:28498]: koniocellular layer of dorsal nucleus of lateral geniculate body'

Are all koniocelluar layers in the dorsal nucleus of the LGB?

**Review by bvarner-ebi — APPROVED** [review]
See note on EA.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28498]: > Are all koniocelluar layers in the dorsal nucleus of the LGB?

yes lol, this took me awhile to get to, but after looking at literature + allen human brain atlas, it is

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:28498]: just to be extra sure, I won't merge till @patrick-lloyd-ray gives a thumbs up :)

Comment (patrick-lloyd-ray) [reply src/ontology/cl-edit.owl:28498]: LGTM.

**Review by patrick-lloyd-ray — APPROVED** [review]

**Pull request merged**